readme.md

```diff
@@ -161,6 +161,7 @@
 ## Text Editor Extensions
 
 - [prolog.vim](https://github.com/adimit/prolog.vim) - Prolog integration for Vim.
+- [repl.vim](https://github.com/ujihisa/repl.vim) - VIM REPL with support for SWIProlog.
```

Comment:
From the [contributing guidelines](https://github.com/sindresorhus/awesome-nodejs/blob/master/contributing.md):

> - Additions should be added to the bottom of the relevant category.

Reply:
Ahm, sorry but, this isn't the relevant category? ._. repl.vim is an extension for the VIM text editor...